### PR TITLE
Revamp ballerina API docs

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: findbugs-exclude.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+    <Match>
+        <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD,BC_UNCONFIRMED_CAST,BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
+    </Match>
+    <Match>
+        <Package name="org.ballerinalang.docgen.model"/>
+        <Package name="org.ballerinalang.docgen.Generator"/>
```
Comment (hemikak) [inline findbugs-exclude.xml]: This should be Class instead of Package.

File: src/main/java/org/ballerinalang/docgen/Generator.java [hunk 2]
```diff
@@ -0,0 +1,459 @@
+package org.ballerinalang.docgen;
```
Comment (hemikak) [inline src/main/java/org/ballerinalang/docgen/Generator.java]: Need copyright header.

File: src/main/java/org/ballerinalang/docgen/Writer.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
+package org.ballerinalang.docgen;
```
Comment (hemikak) [inline src/main/java/org/ballerinalang/docgen/Writer.java]: Need copyright header.

File: src/main/java/org/ballerinalang/docgen/model/ActionDoc.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package org.ballerinalang.docgen.model;
```
Comment (hemikak) [inline src/main/java/org/ballerinalang/docgen/model/ActionDoc.java]: Need copyright header.

File: src/main/java/org/ballerinalang/docgen/model/AnnotationDoc.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package org.ballerinalang.docgen.model;
```
Comment (hemikak) [inline src/main/java/org/ballerinalang/docgen/model/AnnotationDoc.java]: Need copyright header.

File: src/main/java/org/ballerinalang/docgen/model/ConnectorDoc.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
+package org.ballerinalang.docgen.model;
```
Comment (hemikak) [inline src/main/java/org/ballerinalang/docgen/model/ConnectorDoc.java]: Need copyright header.

File: src/main/java/org/ballerinalang/docgen/model/Documentable.java [hunk 7]
```diff
@@ -0,0 +1,31 @@
+package org.ballerinalang.docgen.model;
```
Comment (hemikak) [inline src/main/java/org/ballerinalang/docgen/model/Documentable.java]: Need copyright header.

File: findbugs-exclude.xml (hunk 1 above)
Comment (NatashaWso2) [reply findbugs-exclude.xml]: Fixed with 93bb371

File: src/main/java/org/ballerinalang/docgen/Generator.java (hunk 2 above)
Comment (NatashaWso2) [reply src/main/java/org/ballerinalang/docgen/Generator.java]: Fixed with 93bb371

File: src/main/java/org/ballerinalang/docgen/Writer.java (hunk 3 above)
Comment (NatashaWso2) [reply src/main/java/org/ballerinalang/docgen/Writer.java]: Fixed with 93bb371

File: src/main/java/org/ballerinalang/docgen/model/AnnotationDoc.java (hunk 5 above)
Comment (NatashaWso2) [reply src/main/java/org/ballerinalang/docgen/model/AnnotationDoc.java]: Fixed with 93bb371

File: src/main/java/org/ballerinalang/docgen/model/ActionDoc.java (hunk 4 above)
Comment (NatashaWso2) [reply src/main/java/org/ballerinalang/docgen/model/ActionDoc.java]: Fixed with 93bb371

File: src/main/java/org/ballerinalang/docgen/model/ConnectorDoc.java (hunk 6 above)
Comment (NatashaWso2) [reply src/main/java/org/ballerinalang/docgen/model/ConnectorDoc.java]: Fixed with 93bb371

File: src/main/java/org/ballerinalang/docgen/model/Documentable.java (hunk 7 above)
Comment (NatashaWso2) [reply src/main/java/org/ballerinalang/docgen/model/Documentable.java]: Fixed with 93bb371

**Pull request merged**